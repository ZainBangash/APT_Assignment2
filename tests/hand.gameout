Welcome To Scrabble!
------------------- 
Menu
----
1. New Game
2. Load Game        
3. Credits
4. Quit
1

Starting a New Game

Enter a name for player 1 (uppercase characters only)
TANIA

Enter a name for player 2 (uppercase characters only)
ZAIN


Player 1: TANIA, 0
Player 2: ZAIN, 0

Lets Play!

     1   2   3   4   5   6   7   8   9   10  11  12  13  14  15
  --------------------------------------------------------------
 A |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 B |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 C |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 D |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 E |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 F |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 G |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 H |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 I |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 J |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 K |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 L |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 M |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 N |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 O |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
A-1 X-8 I-1 C-3 H-4 R-1 I-1
Enter command: place H at A1

     1   2   3   4   5   6   7   8   9   10  11  12  13  14  15 
  --------------------------------------------------------------
 A | H |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 B |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 C |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 D |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 E |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 F |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 G |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 H |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 I |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 J |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 K |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 L |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 M |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 N |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 O |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
A-1 X-8 I-1 C-3 R-1 I-1
Enter command: save TEST4


Game saved successfully

Welcome To Scrabble!
-------------------
Menu
----
1. New Game
2. Load Game
3. Credits
4. Quit
2

Enter the filename from which load a game
TEST4
     1   2   3   4   5   6   7   8   9   10  11  12  13  14  15 
  --------------------------------------------------------------
 A | H |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 B |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 C |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 D |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 E |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 F |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 G |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 H |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 I |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 J |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 K |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 L |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 M |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 N |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 O |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
A-1 X-8 I-1 C-3 R-1 I-1 K-5 A-1 X-8 I-1 C-3 R-1 I-1 K-5
Enter command: place O at A4

     1   2   3   4   5   6   7   8   9   10  11  12  13  14  15 
  --------------------------------------------------------------
 A | H |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 B |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 C |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 D |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 E |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 F |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 G |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 H |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 I |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 J |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 K |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 L |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 M |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 N |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 O |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 
TANIA has 4 points

ZAIN has 0 points
 
TANIA has won!

Good Bye