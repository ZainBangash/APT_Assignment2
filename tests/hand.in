1
TANIA
ZAIN
save TEST1
place J at A4