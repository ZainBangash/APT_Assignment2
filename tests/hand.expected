Welcome To Scrabble!
-------------------
Menu
----
1. New Game
2. Load Game
3. Credits
4. Quit
1

Starting a New Game

Enter a name for player 1 (uppercase characters only)
TANIA

Enter a name for player 2 (uppercase characters only)
ZAIN


Player 1: TANIA, 0
Player 2: ZAIN, 0

Lets Play!

     1   2   3   4   5   6   7   8   9   10  11  12  13  14  15
  --------------------------------------------------------------
 A |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 B |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 C |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 D |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 E |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 F |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 G |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 H |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 I |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 J |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 K |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 L |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 M |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 N |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 O |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
O-1 E-1 G-2 E-1 A-1 N-1 E-1
Enter command: save TEST1 


Game saved successfully

Welcome To Scrabble!
-------------------
Menu
----
1. New Game
2. Load Game
3. Credits
4. Quit
2

Enter the filename from which load a game
TEST1
     1   2   3   4   5   6   7   8   9   10  11  12  13  14  15 
  --------------------------------------------------------------
 A |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 B |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 C |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 D |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 E |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 F |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 G |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 H |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 I |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 J |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 K |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 L |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 M |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 N |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 O |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
O-1 E-1 G-2 E-1 A-1 N-1 E-1 O-1 E-1 G-2 E-1 A-1 N-1 E-1
Enter command: place J at A4 

     1   2   3   4   5   6   7   8   9   10  11  12  13  14  15
  --------------------------------------------------------------
 A |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 B |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 C |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 D |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 E |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 F |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 G |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 H |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 I |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 J |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 K |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 L |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 M |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 N |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 O |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 
TANIA has 0 points

ZAIN has 0 points


Good Bye